raysort/main_push.py

```diff
@@ -238,46 +267,44 @@ def final_merge(
     worker_id: PartId,
     reduce_idx: PartId,
     parts: list[np.ndarray],
-    subpart_idx: PartId = 1,
+    parts_size_bytes: int,
 ) -> list[PartInfo]:
     logging_utils.init()
 
     with tracing_utils.timeit("reduce"):
         M = len(parts)
         if M == 0:
             return []
-
-        parts_memory_gb = _get_total_partition_sizes(parts)
+        parts_memory_gb = parts_size_bytes * 1.0 / 1_000_000_000
```

Comment:
I think you can achieve the same by `parts_size_bytes / 1e9`. As long as the second operand is a floating point number, the result will be.